sdk/batch.py

```diff
@@ -91,9 +91,9 @@ def declare_complete(self, wait: bool = False) -> Dict:
         if wait:
             while rsp["status"] in ["PENDING", "RUNNING"]:
                 time.sleep(RESULT_POLLING_INTERVAL)
-                rsp = self._client._get_batch(self.id)
-            jobs_rsp = self._client._get_jobs(self.id, fetch_results=True)
-            for j in jobs_rsp:
-                job = Job(**j)
+                rsp, _ = self._client._get_batch(self.id)
+            for job_id, job in self.jobs.items():
```

Comment:
why not use `fetch_results` arg from `_get_batch` here ?

Reply:
> LGTM! Don't forget to bump the version of the SDK for pypi

I did it already right ? :)

Reply:
Yes sorry, I wasn't looking at all the changes! lgtm